ingest/expression_files/mtx.py

```diff
@@ -142,6 +190,48 @@ def get_features(feature_row: str):
             gene_name = feature_data[1]
         return gene_id, gene_name
 
+    @staticmethod
+    def sort_mtx(file_path) -> str:
+        """
+        Sorts MTX file by gene. File header, dimensions, and comments are not included in sort.
+
+         Returns
+         ----------
+            new_file_path (str) : Full path of newly sorted MTX file
+        """
+        file_name = ntpath.split(file_path)[1]
+        new_file_name = f"{file_name}_sorted_MTX.mtx"
+
+        with open(new_file_name, "w+") as f:
+            GeneExpression.dev_logger.info("Starting to sort")
+            start_time = datetime.datetime.now()
+            # Line to start sorting at
+            start_idx: int = MTXIngestor.get_line_no(file_path)
+            # Only include gene expression data
+            p1 = subprocess.Popen(
+                ["tail", "-n", f"+{start_idx}", f"{file_path}"], stdout=subprocess.PIPE
+            )
+            subprocess.run(
```

Comment:
this could use a comment about what the options are specifying

Reply:
464ec68